### PR TITLE
Fix JSL docs

### PR DESCRIPTION
**Pull request by PhilippeFerreiraDeSousa**

### CODE REVIEW

File: src/layerstack.stanza [hunk 1]
```diff
@@ -487,11 +487,11 @@ layers if any.
 @param index Conductor layer to inspect. This is a zero-based index
 where the top layer is 0, the first inner layer is 1, etc.
 <DOC>
-public defn get-conductor (ls:LayerStack, index:Int) -> [Maybe<LayerSpec>, LayerSpec, Maybe<LayerSpec>] :
+public defn get-conductor (stack:LayerStack, index:Int) -> [Maybe<LayerSpec>, LayerSpec, Maybe<LayerSpec>] :
```
Comment (callendorph) [inline src/layerstack.stanza:490]: we can't use `stack` here because it is a JITX statement keyword.

**Review by callendorph — APPROVED** [review]
:+1: Looks good. 
There is one place to double check - the `stack` keyword I had trouble with so we might want to avoid that.

File: src/layerstack.stanza (hunk 1 above)
Comment (PhilippeFerreiraDeSousa) [reply src/layerstack.stanza:490]: The file is loading up
image

Comment (PhilippeFerreiraDeSousa) [reply src/layerstack.stanza:490]: What is `stack` doing in stanza?

Comment (PhilippeFerreiraDeSousa) [reply src/layerstack.stanza:490]:
```
stanza> val stack = 1
stanza> defn stack (x:Int) : println(3)
stanza> 
stanza> stack(6)
3
false
```

Comment (plasmajitx) [reply src/layerstack.stanza:490]: I think the point is not that it's illegal, but rather that it's potentially confusing and dangerous.

Comment (plasmajitx) [reply src/layerstack.stanza:490]: in esir:
```
  ; stack(thickness, material)
  defrule exp4 = (stack(?thickness:#exp!, ?material:#exp!)) :
    val template = `(make-stackup-layer(JITX-CLIENT, info, false, thickness, material, false))
    ir-fill $ substitute(template, [
      `info => info-exp(closest-info())
      `thickness => thickness
      `material => material])
```

Comment (PhilippeFerreiraDeSousa) [reply src/layerstack.stanza]: Come on, you are made out of sugar xD
I don't mind whatever the name is, code updated.

**Pull request merged**